test/test_tree.cpp

```diff
@@ -848,6 +848,22 @@ void verify_assertion(csubstr src, Function &&fn)
     });
 }
 
+template<class Function>
+void verify_error(Tree &tree, Function &&fn)
+{
+    ExpectError::verify_error(&tree, [&]{
```

Comment:
... just call directly `ExpectError::do_check()` here

Reply:
Yep, updated it to call `do_check()` directly.